smoke/src/main/java/com/adobe/cq/cloud/testing/it/smoke/CreatePageAsAuthorUserIT.java

```diff
@@ -16,48 +16,91 @@
 package com.adobe.cq.cloud.testing.it.smoke;
 
 import com.adobe.cq.testing.client.CQClient;
+import com.adobe.cq.testing.client.CQSecurityClient;
+import com.adobe.cq.testing.client.security.CQAuthorizableManager;
+import com.adobe.cq.testing.client.security.CQPermissions;
+import com.adobe.cq.testing.client.security.Group;
+import com.adobe.cq.testing.client.security.User;
 import com.adobe.cq.testing.junit.rules.CQAuthorClassRule;
-import com.adobe.cq.testing.junit.rules.CQRule;
-import com.adobe.cq.testing.junit.rules.Page;
-import com.adobe.cq.testing.junit.rules.TemporaryUser;
 import org.apache.http.HttpStatus;
 import org.apache.sling.testing.clients.ClientException;
 import org.apache.sling.testing.clients.SlingHttpResponse;
 import org.junit.Assert;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.ClassRule;
-import org.junit.Ignore;
-import org.junit.Rule;
 import org.junit.Test;
-import org.junit.rules.RuleChain;
-import org.junit.rules.TestRule;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import java.util.Date;
 import java.util.UUID;
 
 import static java.util.concurrent.TimeUnit.MINUTES;
 
-@Ignore
+
 public class CreatePageAsAuthorUserIT {
 
+    private static final String CONTENT_NODE = "/content";
     private static final Logger LOG = LoggerFactory.getLogger(CreatePageAsAuthorUserIT.class);
-
     private static final int TIMEOUT = (int) MINUTES.toMillis(2);
 
-    public static final String CONTENT_AUTHORS_GROUP = "content-authors";
-
     @ClassRule
     public static CQAuthorClassRule cqBaseClassRule = new CQAuthorClassRule();
 
-    public CQRule cqBaseRule = new CQRule(cqBaseClassRule.authorRule);
+    private CQAuthorizableManager authorizableManager;
+    private CQClient adminAuthor;
+    private CQClient authorAuthor;
+
+    private CQSecurityClient sClient;
+    private CQPermissions permissionsObj;
+
+    private User testUser;
+    private Group testGroup;
+    private CQClient testUserClient;
 
-    // Create a random page so the test site is initialized properly.
-    private final Page temporaryPage = new Page(cqBaseClassRule.authorRule);
```

Comment:
Why do we need to remove this? We should use Rules and not before and after methods, for modularity.

Reply:
I was wondering how can we create a new user and group and assign correct ACLs using rules. Let me know if there is a way.